e2e/tests/test_31_admin_metrics.py

```diff
@@ -31,28 +31,9 @@ def test_metrics() -> None:
 
     metric_names = set(metrics.keys())
 
-    # the cache and queue metrics are computed by the background jobs. Here, in the e2e tests, we don't run them,
```

Comment:
how could we test in e2e that we have the metrics?

Reply:
Actually, since this test runs after `test_11_api.py`, some of the cache records should have already been computed so, we could simply change the not has_metric logic to the opposite.